### PR TITLE
remove instance_type parameter in renew_task, add login info

### PR DESCRIPTION
**Pull request by alphaflows**

### CODE REVIEW

**Comment by alphaflows** [comment]
- remove `instance_type` parameter requirement in `renew_task`, `renew_payment`, `submit_payment` functions in orchestrator

**Comment by alphaflows** [comment]
- remove `repo_name`, `repo_owner`, `repo_branch` parameters
- change `auto_pay` default to True to simplify logic
- add `INFO - Logging in Testnet/Mainnet` log info

File: swan/api/orchestrator.py [hunk 1]
```diff
@@ -633,23 +620,24 @@ def renew_task(
             duration: duration of service runtime (seconds).
             tx_hash: (optional)tx_hash of submitted payment
             private_key: (required if no tx_hash)
-            auto_pay: (required True if no tx_hash and private_key)
+            auto_pay: (required True if no tx_hash but with private_key provided)
```
Comment (PlutoNbai) [inline swan/api/orchestrator.py:623]: Consider removing this, auto_pay should be true

File: swan/session.py [hunk 2]
```diff
@@ -32,8 +32,10 @@ def __init__(
             self.login_url = login_url
         elif network == "testnet":
             self.login_url = ORCHESTRATOR_API_TESTNET
+            logging.info("Logging in Testnet")
         else:
             self.login_url = ORCHESTRATOR_API_MAINNET
+            logging.info("Logging in Mainnet")
```
Comment (PlutoNbai) [inline swan/session.py:38]: Can you double check if you remove the logs when user logged in, as a user, I don't wanna see this log twice

**Review by PlutoNbai — CHANGES_REQUESTED** [review]

File: docs/key_functions.md [hunk 3]
```diff
@@ -131,7 +125,7 @@ response = swan_orchestrator.renew_task(
   task_uuid="string"
   duration=3600, 
   tx_hash="string", 
-  auto_pay = False, 
+  auto_pay = True,
```
Comment (sonic-chain) [inline docs/key_functions.md:128]: You can delete `auto_pay`. If `private_key` has a value, it means automatic payment.

File: swan/api/orchestrator.py [hunk 4]
```diff
@@ -465,21 +450,22 @@ def estimate_payment(self, duration: float = 3600, instance_type: str = None):
             logging.error(str(e) + traceback.format_exc())
             return None
     
-    def submit_payment(self, task_uuid, private_key, duration = 3600, instance_type = None):
+    def submit_payment(self, task_uuid, private_key, duration = 3600, **kwargs):
```
[the remark this answers is not in the document]
Comment (sonic-chain) [reply swan/api/orchestrator.py:458]: Only supports automatic payment ？I don't see the manual payment function.

**Comment by alphaflows** [comment]
add `repo_branch` back

File: docs/key_functions.md [hunk 5]
```diff
@@ -71,12 +69,10 @@ PARAMETERS:
 - **duration** (integer) - duration of service runtime in seconds. Defaults to 3600 seconds (1 hour).
 - **app_repo_image** (string) - The name of a demo space. If app_repo_image is used, auto_pay will be True by default. To learn more about auto_pay, check out auto_pay parameter. If you want turn auto_pay off, set auto_pay to False
 - **job_source_uri** (string) - The job source URI to be deployed. If this is provided, app_repo_image and repo_uri are ignored. The repository must contain a dockerfile
-- **repo_uri** (string) - The The URI of the repo to be deployed. The repository must contain a dockerfile \
+- **repo_uri** (string) - The The URI of the repo to be deployed. The repository must contain a dockerfile. Please see [repo_uri](repo_uri.md)
```
Comment (Normalnoise) [inline docs/key_functions.md:72]: - repo_uri: The repository must contain a `Dockerfile` or `deploy.yaml`; if there are `Dockerfile` or `deploy.yaml` in the repo, your project will be deployed by `deploy.yaml`, `Dockerfile ` will be ignored

 - `deploy.yaml` must be written according to [LDL specifications](https://docs.lagrangedao.org/spaces/intro/lagrange-definition-language-ldl)

**Pull request merged**